### PR TITLE
Add system utils needed for 'remote process lookup' Agent functionality

### PR DESCRIPTION
**Pull request by mdonkers**

### CODE REVIEW

**Comment by htreu** [comment]
This supersedes #51.

File: rhel/Dockerfile [hunk 1]
```diff
@@ -57,12 +57,7 @@ RUN curl -sSL https://packages.instana.io/Instana.gpg -o /tmp/Instana.gpg && \
     echo "256bd490a55a1939a4c9cd15c043404b79a86429ee04129c00d33dab8c0cf040 /usr/bin/runc" | sha256sum -c - > /dev/null && \
     chmod 755 /usr/bin/runc && \
     rm -rf /tmp/* /etc/yum.repos.d/Instana-Agent.repo /etc/yum.repos.d/docker.repo && \
-    microdnf clean all && \
-    curl https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 -o /usr/bin/jq && \
```
Comment (htreu) [inline rhel/Dockerfile:61]: **question:** can we safely ignore jq & pip during installation?

File: static/Dockerfile.s390x_gomplate [hunk 2]
```diff
@@ -1,8 +1,8 @@
-FROM s390x/ubuntu:18.04
+FROM s390x/ubuntu:20.04
```
Comment (htreu) [inline static/Dockerfile.s390x_gomplate:1]: **question:** why not also build gomplate on 21.04?

**Review by htreu — COMMENTED** [review]
looks good, 2 questions inline

File: static/Dockerfile.s390x_gomplate (hunk 2 above)
Comment (mdonkers) [reply static/Dockerfile.s390x_gomplate:1]: forgot, will change

File: rhel/Dockerfile (hunk 1 above)
Comment (htreu) [reply rhel/Dockerfile:61]: pip is noe in python3, jq already installed above.

Comment (mdonkers) [reply rhel/Dockerfile:61]: `python` + `python-setuptools` + `pip` (manually installed) got superseded by just `python3`, already including all the other necessary dependencies.
`jq` is installed a few lines above already, so this was obsolete.

**Pull request merged**